crates/cast/src/starknet_commands/script.rs

```diff
@@ -350,20 +348,19 @@ impl CairoHintProcessor<'_> {
 
 pub fn run(
     module_name: &str,
-    path_to_scarb_toml: &Option<Utf8PathBuf>,
     provider: &JsonRpcClient<HttpTransport>,
     runtime: Runtime,
     config: &CastConfig,
+    contracts: &HashMap<String, StarknetContractArtifacts>,
```

Comment:
Looking at the type it would suggest that this hashmap stores the mapping to contracts. I assume it `StarknetContractArtifacts` was reused to store the compiled script code too? If that's the case it should be refactored